packages/edit-site/src/components/global-styles/font-library-modal/library-font-variant.js

```diff
@@ -20,17 +20,18 @@ function LibraryFontVariant( { face, font } ) {
 	const { isFontActivated, toggleActivateFont } =
 		useContext( FontLibraryContext );
 
-	const isIstalled = font?.fontFace
-		? isFontActivated(
-				font.slug,
-				face.fontStyle,
-				face.fontWeight,
-				font.source
-		  )
-		: isFontActivated( font.slug, null, null, font.source );
+	const isIstalled =
```

Comment:
```suggestion
	const isInstalled =
```

Reply:
Also below, on line 55.

Reply:
Quick PR to fix these typos: https://github.com/WordPress/gutenberg/pull/58243